migrations/2_pool_impl.js

```diff
@@ -11,43 +10,59 @@ var DelegatedDepositVerifierStage = artifacts.require("DelegatedDepositVerifierS
 var ZkBobPoolERC20 = artifacts.require("ZkBobPoolERC20");
 
 module.exports = async function(deployer) {
-    const usdt = TronWeb.address.toHex(process.env.TOKEN);
+    const usdt = TronWeb.address.fromHex(process.env.TOKEN);
+    // 410000000000000000000000000000000000000000 does not work for some reason
+    const zeroAddress = TronWeb.address.fromHex('410000000000000000000000000000000000000001');
```

Comment:
```suggestion
    const zeroAddress = '410000000000000000000000000000000000000000';
```
might work, it's what is being used in the 4_token_seller script.